Android-Sample/README.md

```diff
@@ -30,25 +30,25 @@ To configure the app:
 2. Sync Gradle
 
 3. In the com.opentok.qualitystats.sample.MainAcdtivity.java class, set the following properties
-   to a test OpenTok session ID, token, and API key:
+   to a test Vonage Video API session ID, token, and application ID:
 
    ```
    private static final String SESSION_ID = "";
    private static final String TOKEN = "";
-   private static final String APIKEY = "";
+   private static final String APPLICATION_ID = "";
    ```
 
    **Important:** You must use a unique session, with its own session ID, for the network test. This
    must be a different session than the one you will use to share audio-video streams between
    clients. Do not publish more than one stream to the test session.
 
    The app requires that each session uses the routed media mode -- one that uses
-   the [OpenTok Media Router](https://tokbox.com/developer/guides/create-session/#media-mode).
+   the [Vonage Video API Media Router](https://tokbox.com/developer/guides/create-session/#media-mode).
```

Comment:
Please change all docs link URLs throughout to point to the Vonage version of the docs. For example, this one should point to https://developer.vonage.com/en/video/guides/create-session#the-media-router-and-media-modes.